lib/patch_ruby/models/create_order_request.rb

```diff
@@ -232,7 +232,7 @@ def valid?
       return false if !@mass_g.nil? && @mass_g > 100000000000
       return false if !@mass_g.nil? && @mass_g < 0
       return false if !@total_price_cents_usd.nil? && @total_price_cents_usd < 1
-      state_validator = EnumAttributeValidator.new('String', ["draft", "placed"])
+      state_validator = EnumAttributeValidator.new('String', ["draft", "reserved", "placed"])
```

Comment:
@joustrich Is introducing this okay?

Reply:
On behalf of @joustrich, I'll say yes 😁

Reply:
Shoot sorry I totally missed this. Yes we do want this in there!